### DISCUSSION TITLE
Inverting Calibration Correction Terms for S-Parameter Measurements

### DISCUSSION THREAD

**Discussion by jobonny** [scikit-rf/scikit-rf#1214] [Q&A] [answered]
Hello everyone,

I am currently assessing the effect of different calibration methods on S-parameter measurements performed on-wafer.
I have collected raw measurements of the standards used during calibration, as well as the error terms computed by WinCal and applied to the VNA.

Is there an easy way to invert these correction terms to recover the raw measurements?

Thank you for your help!

Giorgio

**Comment by arsenovic** [comment, marked as answer]
https://scikit-rf.readthedocs.io/en/latest/tutorials/Calibration.html

lets us know specific questions you have.
